bundles/route/index.rst

```diff
@@ -140,26 +140,26 @@ RouteDefaultsProvider
 
 .. code-block:: php
 
-    namespace AppBundle\Routing;
+    namespace App\Routing;
 
-    use AppBundle\Entity\Recipe;
-    use AppBundle\Entity\RecipeRepository;
+    use App\Entity\Event;
+    use App\Entity\EventRepository;
     use Sulu\Bundle\RouteBundle\Routing\Defaults\RouteDefaultsProviderInterface;
 
-    class RecipeRouteDefaultProvider implements RouteDefaultsProviderInterface
+    class EventRouteDefaultProvider implements RouteDefaultsProviderInterface
     {
-        protected $recipeRepository;
+        protected $eventRepository;
 
-        public function __construct(RecipeRepository $recipeRepository)
+        public function __construct(EventRepository $eventRepository)
         {
-            $this->recipeRepository = $recipeRepository;
+            $this->eventRepository = $eventRepository;
         }
 
         public function getByEntity($entityClass, $id, $locale, $object = null)
         {
             return [
-                '_controller' => 'AppBundle:Recipe:index',
-                'recipe' => $object ?: $this->recipeRepository->find($id, $locale),
+                '_controller' => 'App:Event:index',
```

Comment:
here we should go with FQCN as it is documented in the symfony docs for routing now:

```php
'_controller' => 'App\Controller\EventController::indexAction',
```